runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkExecutionEnvironments.java

```diff
@@ -171,4 +176,12 @@ public static StreamExecutionEnvironment createStreamExecutionEnvironment(
 
     return flinkStreamEnv;
   }
+
+  private static void applyLatencyTrackingInterval(
+      ExecutionConfig config, FlinkPipelineOptions options) {
+    long latencyTrackingInterval = options.getLatencyTrackingInterval();
+    if (latencyTrackingInterval != -1) {
```

Comment:
nit: `latencyTrackingInterval > 0`

Reply:
latencyTrackingINteraval = 0 will disable the feature. This is what I need (reason is `FLINK-10226`).
The idea is (taken from other options above), is user specify something, forward it to flink. Reasonable values are `interval >= 0` but flink accepts negative numbers and threat them as `0` = disable.
I can do `latencyTrackingInterval > 0` if you consider it more fit. Should I?

Reply:
> 0 will be more explicit so lets go with that.